shared/src/bundles/users/enums/user-validation-message.enum.ts

```diff
@@ -3,6 +3,7 @@ enum UserValidationMessage {
     LASTNAME_REQUIRE = 'Last name is required',
     EMAIL_REQUIRE = 'Email is required',
     EMAIL_WRONG = 'Email is wrong',
+    EMAIL_INVALID = 'Invalid email format. Please use a valid email address',
```

Comment:
what's difference between EMAIL_WRONG and EMAIL_INVALID? What purpose of the first one?

Reply:
email invalid for custom validation error. wrong - if it doesn't have email format

Reply:
as I see from the message EMAIL_INVALID tells user about **Invalid email format.** so why do we need both?
can you provide real example when user get each of them?

Reply:
see specification and MY changes.